### PR TITLE
MTM-57487-Unify-usage-of-select-vs-choose

### PR DESCRIPTION
**Pull request by dianaolives**
Replace usage of “choose” with “select” wherever this makes sense (is replaceable without changing the meaning).

### CODE REVIEW

**Review by MWindel — APPROVED** [review]

File: content/data-broker/ms-data-broker-bundle/using-ms-data-broker.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ The microservice-based data broker is powered by the {{< product-c8y-iot >}} Mes
 
 {{< c8y-admon-req >}}
 The {{< product-c8y-iot >}} Messaging Service is an optional component of the {{< product-c8y-iot >}} platform that may need to be enabled before the microservice-based data broker can be used.
-The original data broker will continue to operate alongside the microservice-based data broker for the time being, and users can choose which data broker to use on a per-tenant basis.
+The original data broker will continue to operate alongside the microservice-based data broker for the time being, and users can select which data broker to use on a per-tenant basis.
```
Comment (BeateRixen) [inline content/data-broker/ms-data-broker-bundle/using-ms-data-broker.md:13]: I´d say, here "choose" is correct. It is "choose between options" which we should leave. What do you think, @MWindel ?

File: content/datahub/setting-up-datahub-bundle/setting-up-initial-configuration.md [hunk 2]
```diff
@@ -39,7 +39,7 @@ When using the proxy REST API, all queries are processed using the same Dremio A
 Your follow-up application might require more than one Dremio user for accessing the data lake. You can define additional Dremio users for that purpose, using the instructions in [Adding a Dremio user](/datahub/setting-up-datahub/#adding-a-dremio-user).
 
 #### Data Lake {#data-lake}
-Depending on the configuration of the environment, the data lake provider is either fixed or you can choose among different providers. For each data lake provider, you must specify corresponding settings to define the data lake to be used.
+Depending on the configuration of the environment, the data lake provider is either fixed or you can select among different providers. For each data lake provider, you must specify corresponding settings to define the data lake to be used.
```
Comment (BeateRixen) [inline content/datahub/setting-up-datahub-bundle/setting-up-initial-configuration.md:42]: Same here: It is "choose among options". "Select" sounds weird here.

File: content/microservice-sdk/http-bundle/node-js-microservice.md [hunk 3]
```diff
@@ -4,7 +4,7 @@ title: Node.js microservice
 layout: redirect
 ---
 
-{{< product-c8y-iot >}} provides an SDK for developing microservices using Java. Nevertheless, you are free to choose the tech-stack of your preference to develop a microservice as long as it fulfills the [general requirements](/microservice-sdk/general-aspects/#requirements).
+{{< product-c8y-iot >}} provides an SDK for developing microservices using Java. Nevertheless, you are free to select the tech-stack of your preference to develop a microservice as long as it fulfills the [general requirements](/microservice-sdk/general-aspects/#requirements).
```
Comment (BeateRixen) [inline content/microservice-sdk/http-bundle/node-js-microservice.md:7]: Same here: Better leave "choose"

File: content/microservice-sdk/java-bundle/developing-microservice.md [hunk 4]
```diff
@@ -30,13 +30,13 @@ Annotation | Description
 
 ### Context support {#context-support}
 
-The context support is covered by the annotation `@EnableContextSupport`. It allows to choose between `@TenantScope` and `@UserScope` which is related to the user management of microservices, as described in [General aspects](/microservice-sdk/general-aspects) in {{< product-c8y-iot >}}.
+The context support is covered by the annotation `@EnableContextSupport`. It allows to select between `@TenantScope` and `@UserScope` which is related to the user management of microservices, as described in [General aspects](/microservice-sdk/general-aspects) in {{< product-c8y-iot >}}.
```
Comment (BeateRixen) [inline content/microservice-sdk/java-bundle/developing-microservice.md:33]: better "choose between"

File: content/data-broker/ms-data-broker-bundle/using-ms-data-broker.md (hunk 1 above)
Comment (MWindel) [reply content/data-broker/ms-data-broker-bundle/using-ms-data-broker.md:13]: @dianaolives @BeateRixen I agree. 

```suggestion
The original data broker will continue to operate alongside the microservice-based data broker for the time being, and users can choose which data broker to use on a per-tenant basis.
```

**Review by MWindel — APPROVED** [review]

**Pull request merged**